code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -135,6 +135,11 @@
 	crusher_drop_mod = 20
 	dwarf_mob = TRUE
 
+/mob/living/simple_animal/hostile/asteroid/hivelord/legion/beegion
+	name = "beegion"
+	desc = "You can still see what was once a human under the shifting mass of - oh my God, those are bees."
+	brood_type = /mob/living/simple_animal/hostile/poison/bees/toxin
```

Comment:
Bees would attack the legion before the miner here.
Maybe make the bees be lava land themed and have all the normal protection a lava land mob gets.

Reply:
Nvm Im bad at reading vars